datastore/cloud-client/snippets.py

```diff
@@ -916,7 +916,7 @@ def regional_endpoint():
     from google.cloud import datastore
     from google.api_core.client_options import ClientOptions
 
-    ENDPOINT = "https://nam5-datastore.googleapis.com"
+    ENDPOINT = "https://datastore.africa-south1.rep.googleapis.com"
```

Comment:
Is there a reason why these are changing from North America multi-regional to Johannesburg regional? Is it because we don't want to use us-central1 as default?